pkg/sql/opt/invertedidx/inverted_index_expr.go

```diff
@@ -81,6 +82,8 @@ func TryFilterInvertedIndex(
 ) {
 	// Attempt to constrain the prefix columns, if there are any. If they cannot
 	// be constrained to single values, the index cannot be used.
+	// TODO(mgartner): Use computedColumns to attempt to constrain prefix
+	// columns.
```

Comment:
I _think_ we're already doing this with the `optionalFilters`. Leaving this note to remind me to verify.